client/src/app/modules/welcome/welcome.component.scss

```diff
@@ -0,0 +1,79 @@
+$orange-color: #F5612C;
+$blue-color: #2DADB7;
+.welcome-body {
+    height: 100vh;
+    width: 100%;
+    background-image: url(../../../assets/images/background/welcome-background.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.website-title {
+    font-family: cursive;
+    font-size: 3vw;
+    font-weight: bold;
+    color: $orange-color;
+    width: fit-content;
+    position: fixed;
+    z-index: 2;
+    margin-top: 3%;
+    margin-left: 8%;
+}
+
+.motto {
+    font-family: cursive;
+    font-size: 2vw;
+    color: $orange-color;
+    width: fit-content;
+    position: fixed;
+    z-index: 2;
+    margin-top: 7%;
+    margin-left: 3%;
+}
+
+.button {
+    font-family: cursive;
+    position: fixed;
+    z-index: 2;
```

Comment:
please make sure z-index is necessary!

Reply:
They were not necessary so I removed all of them